Lib/defconQt/fontView.py

```diff
@@ -330,25 +361,31 @@ def writeUnicodes(self):
         self._blocked = False
 
     def writeWidth(self):
-        if self._glyph is None: return
+        if self._glyph is None:
+            return
         self._blocked = True
         self._glyph.width = int(self.widthEdit.text())
         self._blocked = False
 
     def writeLeftSideBearing(self):
-        if self._glyph is None: return
+        if self._glyph is None:
+            return
         self._blocked = True
         self._glyph.leftMargin = int(self.leftSideBearingEdit.text())
         self._blocked = False
 
     def writeRightSideBearing(self):
-        if self._glyph is None: return
+        if self._glyph is None:
+            return
         self._blocked = True
         self._glyph.rightMargin = int(self.nameEdit.text())
         self._blocked = False
 
 # TODO: implement Frederik's Glyph Construction Builder
```

Comment:
Is weird to have this comment in the middle of nowhere... maybe move it before the init function.